h/static/styles/partials-v2/_search.scss

```diff
@@ -49,35 +46,55 @@
   }
 }
 
+.search-result-bucket__header {
+  padding: 10px 10px;
+
+  display: flex;
+  flex-direction: row;
+
+  cursor: pointer;
+  user-select: none;
```

Comment:
Not sure we need to prevent the user from selecting the text in the header, do we? This seems like it could be super annoying.